assets/sass/vendor/_mdc-dialog.scss

```
@@ -138,3 +138,13 @@
 		}
 	}
 }
+
+.googlesitekit-settings-module--reader-revenue-manager .mdc-dialog {
```

We should apply these changes to the disconnect modal for all modules for consistency, e.g. by adding a `className` to the `ModalDialog` in `ConfirmDisconnect` (say `googlesitekit-settings-module__confirm-disconnect-modal`), and changing this selector to use that classname.

https://github.com/google/site-kit-wp/blob/7d28e8f2b0c7cbd9398fad57f41dcdd2d8dde923/assets/js/components/settings/SettingsActiveModule/ConfirmDisconnect.js#L161

```suggestion
.googlesitekit-settings-module__confirm-disconnect-modal .mdc-dialog {
```